packages/web/src/pages/_app.tsx

```diff
@@ -28,7 +24,7 @@ import GnoswapServiceProvider from "@providers/gnoswap-service-provider/GnoswapS
 import GnoswapThemeProvider from "@providers/gnoswap-theme-provider/GnoswapThemeProvider";
 import SnackbarProvider from "@providers/snackbar-provider/SnackbarProvider";
 
-import nextI18NextConfig from "../next-i18next.config.js";
+import nextI18NextConfig from "../../next-i18next.config.js";
```

Comment:
Fixed by using absolute path instead of relative path:

before:
```typescript
import nextI18NextConfig from "../next-i18next.config.js";
```

after:
```typescript
import nextI18NextConfig from "next-i18next.config.js";
```